entities/producer.py

```diff
@@ -14,17 +15,21 @@ def __init__(self, env, region):
 
         self.region = region
         self.location = region.draw_location()
-
-    def produce(self):
+        self.producer_id = 0
+    def produce(self, region):
         shipments = []
 
         for _ in range(self.production_rate):
             shipment = Shipment(producer_id=self.registration_id,
                                 location=self.location,
-                                destination=self._set_destination())
+                                destination=self._set_destination(region))
             shipments.append(shipment)
 
         return shipments
 
-    def _set_destination(self):
-        pass  # TODO
+    def _set_destination(self, region):
+        shipment_destination = {"coordinates": region.draw_location(),
```

Comment:
This is not the right format for a shipment_destination: coordinates and region idea _cannot_ be separate (as a geography is associated with a region object)